src/api.rs

```diff
@@ -578,6 +615,7 @@ impl<T: Pixel> Context<T> {
     ))
   }
 
+  /// Send the details of the frame by passing the frame details as input
```

Comment:
Send a frame to be encoded.

Mention that the function may return an error if the input queue is full.